pkg/ui/src/views/statements/barCharts.tsx

```diff
@@ -227,6 +228,40 @@ export function rowsBreakdown(s: StatementStatistics) {
   };
 }
 
+
+export function genericBarChart(s: NumericStat, count: number | Long) {
+    const mean = s.mean;
+    const sd = stdDevLong(s, count);
+
+    const max = mean + sd;
+    const scale = d3.scale.linear()
+      .domain([0, max])
+      .range([0, 100]);
+    return function MakeGenericBarChart() {
+      const width = scale(clamp(mean - sd));
+      const right = scale(mean);
+      const spread = scale(sd + (sd > mean ? mean : sd));
+      const title = renderNumericStatLegend(count, mean, sd, d3.format(".2f"));
+      return (
+        <ToolTipWrapper text={ title } short>
+          <div className={cx("bar-chart", "bar-chart--breakdown")}>
+            <div className={cx("bar-chart__label")}>{ mean }</div>
```

Comment:
Wrap `mean` in the formatter

Reply:
To assist with building bits and pieces like this, we're trying to get more components in a state where they can be rendered independently. This is possible with the bar charts!

Run
`yarn run storybook` in `pkg/ui` and add an entry to the `barCharts.stories.tsx` file like this:

```
  .add("genericBarChart", () => {
    const stat = new NumericStat();
    stat.mean = 25;
    stat.squared_diffs = 25;
    const barChartFactory = genericBarChart(stat, Long.fromInt(10));
    return barChartFactory();
  });
```

Then when you navigate to `localhost:9009` you can see a new bar chart story with your entry and you can tweak and see what it will look like under a bunch of different inputs. Hopefully that can help with development.

Reply:
Perfect! Thank you!